Weather App/Common/Constants.swift

```diff
@@ -0,0 +1,12 @@
+
+import UIKit
+
+let normalFontSize : CGFloat = 30.0
```

Comment:
To whom these constants belong to?

Reply:
No one, I will put the constant in places it belongs.